templates/haproxy-config

```diff
@@ -22,6 +23,9 @@ case "$1" in
               cp "$conf_fragment" "{{ LOAD_BALANCER_CONF_DIR }}/$fragment_name"
         flock -s "{{ LOAD_BALANCER_BACKENDS_DIR }}" \
               awk '{print tolower($1) " " $2}' "$map_fragment" > "{{ LOAD_BALANCER_BACKENDS_DIR }}/$fragment_name"
+        if [ -x "$(command -v etckeeper)" ]; then
+            etckeeper commit "Auto-commit after `haproxy-config apply`."
```

Comment:
This shouldn't be back ticks, since this results in haproxy-config apply actually being called, and the output of the call being included in the commit message.  Sorry, this was my bad – I accidentally used Markdown in an inappropriate place.  Here's a version with correct quoting:

    etckeeper commit 'Auto-commit after "haproxy-config apply".'

Reply:
Ah! Snap, thanks for noticing.